proposals/0126-atf-additional-transports.md

```diff
@@ -33,27 +33,28 @@ Main reasons :
 ## Proposed solution
 
 Use real mobile device as a transport adapter.
-Create Mobile application as a part of ATF infrastructure. 
-Mobile application may use [SDL android library](https://github.com/smartdevicelink/sdl_android)
-for communicating with SDL and TCP connection for communication with ATF.
-
-Mobile application (Mobile transport adapter) should provide to ATF side such RPCs:
- - ConnectToSDL(transport)
- - RemoveConnection(connection_id)
- - GetListOfAvailableTransports()
- - SendData(connection_id, data)
- 
-ATF should be able to connect to multiple Mobile transport adapters and provide ability to use any of them to test engineer. 
-
-ATF should provide such API to test engineer: 
- - ConnectToSDL(device, connection type)
- - RemoveConnection(connection)
- - SendRawData(connection)
- - GetListOfAvailableDevices()
- - GetDeviceInfo(device)
- 
-_СonnectToSDL_ should return connection object.
-Connection object should provide ability to create sessions on it. Session interface should not be changed.  
+Create Android mobile application as a part of ATF infrastructure.
+
+Mobile application should use HTTP connection with REST protocol for control communication with ATF using next API:
+ - GetDeviceInfo - Provides device's information
+ - ConnectToSDL - Connects to SDL and provides proxy TCP connection parameters
+ - DisconnectFromSDL - Disconnects from SDL
+ - GetSDLConnectionStatus - Provides status of connection to SDL
+
+Mobile application should provide next information as main part of response to GetDeviceInfo request:
+ - Operating system
+ - MAC address
```

Comment:
@aderiabin I believe MAC address also should be an array OR moved into an array of available transports like a sub param because each transport chipset of the device [may have](https://stackoverflow.com/a/55993191) its own MAC address so it's not a unique characteristic of the phone itself. For example, you can use your phone name (Settings -> About -> Device name) if you need some unique attribute for sorting.

Reply:
@AKalinich-Luxoft  Agree. In additional to it, Android removes programmatic access to the device’s local hardware identifier (starting from Android 6.0). It would be work only for old devices. 
[doc](https://developer.android.com/about/versions/marshmallow/android-6.0-changes#behavior-hardware-id)

Reply:
@AKalinich-Luxoft, @YaroslavLutsenko Parameter `MAC address has been removed. Parameter `Array of transport` has been extended by transport specific information 
(see 36b2cdb)